nym-vpn-apple/Services/Sources/Services/ConfigurationManager/ConfigurationManager.swift

```diff
@@ -1,54 +1,53 @@
 import Foundation
 import Constants
 
-public final class ConfigurationManager {
-    public static func configureMainnetEnvironmentVariables() {
-        setenv("CONFIGURED", "true", 1)
-        setenv("RUST_LOG", "info", 1)
-        setenv("RUST_BACKTRACE", "1", 1)
-        setenv("NETWORK_NAME", "mainnet", 1)
-        setenv("BECH32_PREFIX", "n", 1)
-        setenv("MIX_DENOM", "unym", 1)
-        setenv("MIX_DENOM_DISPLAY", "nym", 1)
-        setenv("STAKE_DENOM", "unyx", 1)
-        setenv("STAKE_DENOM_DISPLAY", "nyx", 1)
-        setenv("DENOMS_EXPONENT", "6", 1)
-
-        setenv("REWARDING_VALIDATOR_ADDRESS", "n10yyd98e2tuwu0f7ypz9dy3hhjw7v772q6287gy", 1)
-        setenv("MIXNET_CONTRACT_ADDRESS", "n17srjznxl9dvzdkpwpw24gg668wc73val88a6m5ajg6ankwvz9wtst0cznr", 1)
-        setenv("VESTING_CONTRACT_ADDRESS", "n1nc5tatafv6eyq7llkr2gv50ff9e22mnf70qgjlv737ktmt4eswrq73f2nw", 1)
-        setenv("GROUP_CONTRACT_ADDRESS", "n1e2zq4886zzewpvpucmlw8v9p7zv692f6yck4zjzxh699dkcmlrfqk2knsr", 1)
-        setenv("MULTISIG_CONTRACT_ADDRESS", "n1txayqfz5g9qww3rlflpg025xd26m9payz96u54x4fe3s2ktz39xqk67gzx", 1)
-        setenv("COCONUT_DKG_CONTRACT_ADDRESS", "n19604yflqggs9mk2z26mqygq43q2kr3n932egxx630svywd5mpxjsztfpvx", 1)
+public enum ConfigurationManager {
+    public static func setEnvVariablesFor(environment: Env) throws {
```

Comment:
Nit: I think modern swift prefers the following naming: `setEnvVariables(for environment: Env) throws`